### PR TITLE
Add how-to for reinitializing storage disk on Exoscale

### PR DESCRIPTION
**Pull request by simu**
- [x] Rebase after #98 is merged

Tested multiple times without encountering any problems.

### CODE REVIEW

File: docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+
+== Starting situation
+
+* You already have a OpenShift 4 cluster on Exoscale
+* You have admin-level access to the cluster
+* You want to reinitialize a corrupted storage disk of an existing storage node of the cluster.
+
+== Prerequisites
+
+The following CLI utilities need to be available locally:
```
Comment (ccremer) [inline docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc:16]: What are the typical symptoms of a corrupted disk? How would I verify that I have such a case and should follow this guide?

**Review by ccremer — APPROVED** [review]

File: docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc (hunk 1 above)
Comment (simu) [reply docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc]: added a tip box describing the main symptoms indicating a corrupted storage disk

File: docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc [hunk 2]
```diff
@@ -0,0 +1,229 @@
+= Reinitialize a storage disk
+
+[abstract]
+--
+Steps to reinitialize an existing, but corrupted storage disk of an OpenShift 4 cluster on https://www.exoscale.com[Exoscale].
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc:5]:
```suggestion
Steps to reinitialize an existing, but corrupted Ceph storage disk of an OpenShift 4 cluster on https://www.exoscale.com[Exoscale].
```

File: docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+--
+
+== Starting situation
+
+* You already have a OpenShift 4 cluster on Exoscale
+* You have admin-level access to the cluster
+* You want to reinitialize a corrupted storage disk of an existing storage node of the cluster.
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc:12]:
```suggestion
* You want to reinitialize a corrupted Ceph storage disk of an existing storage node of the cluster.
```

File: docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc [hunk 4]
```diff
@@ -0,0 +1,229 @@
...
+
+The following CLI utilities need to be available locally:
+
+* `kubectl`
+
+== Reinitialize storage disk
+
+=== Gather information
+
+. Make a note of the ID of OSD of the disk you want to reinitialize
```
Comment (corvus-ch) [inline docs/modules/ROOT/pages/how-tos/exoscale/reinitialize_storage_disk.adoc:32]:
```suggestion
. Make a note of the OSD ID for the disk you want to reinitialize
```

of of of … ;-)

**Review by corvus-ch — APPROVED** [review]
Some minor remarks on wording. Apart from that LGTM.

I realy like where this is going with all those includes.

**Pull request merged**